packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -41,8 +41,11 @@ test.describe('Call Composite E2E Tests', () => {
       expect(await page.screenshot()).toMatchSnapshot(`page-${idx}-local-device-settings-camera-disabled.png`);
       await page.click(dataUiId('call-composite-local-device-settings-microphone-button'));
       await page.click(dataUiId('call-composite-local-device-settings-camera-button'));
-      await page.waitForSelector('video');
-      await page.waitForTimeout(1000);
+      await page.waitForFunction(() => {
+        const videoNode = document.querySelector('video');
+        const videoLoaded = videoNode?.readyState === 4;
```

Comment:
Tangential: I don't like the fact that the test app and the tests are very intertwined :/